<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numPad</class>
 <widget class="QWidget" name="numPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>numPad</string>
  </property>
  <widget class="MPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>151</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_point">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Backspace">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删  除</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Enter">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确  认</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_oem1">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_plus">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_minus">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_oem2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_oem3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;&amp;</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_multi">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_oem4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>150</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>@</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_div">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_space">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>151</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>空 格</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_enPad">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>151</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>英文键盘</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
